programs/vote/src/vote_state/mod.rs

```diff
@@ -586,12 +586,17 @@ pub fn update_node<S: std::hash::BuildHasher>(
 ) -> Result<(), InstructionError> {
     let mut vote_state: VoteState =
         State::<VoteStateVersions>::state(vote_account)?.convert_to_current();
-    let authorized_voter = vote_state
+
+    // Allow only one update per epoch
+    let _authorized_voter = vote_state
```

Comment:
@carllin - can you please review this function in particular

Reply:
No test fails when I remove the call to `get_and_update_authorized_voter()`, so that seems a little problematic.

Reply:
@mvines this is safe to remove!

Reply:
`test_vote_update_node_id` should fail if you remove the  two lines `get_and_update_authorized_voter()` and `verify_authorized_signer(&authorized_voter, signers)?;`

Reply:
Ok, thanks.  

@carllin, so this is what happens when I remove - c0d89bdb787aee842e76710a96efa1cdddba1da4.  No more need for Clock in UpdateNode!

Reply:
👍but we love clocks xD